x-pack/plugins/cloud_security_posture/public/components/accounts_evaluated_widget.tsx

```diff
@@ -33,8 +33,8 @@ export const AccountsEvaluatedWidget = ({
   const cisAwsClusterAmount = filterClustersById(CIS_AWS).length;
   const cisGcpClusterAmount = filterClustersById(CIS_GCP).length;
 
-  const cisAwsBenchmarkName = filterClustersById(CIS_AWS)[0]?.meta.benchmark.name || '';
-  const cisGcpBenchmarkName = filterClustersById(CIS_GCP)[0]?.meta.benchmark.name || '';
+  const cisAwsBenchmarkName = 'Amazon Web Services (AWS)';
```

Comment:
Should these be i18n?

Reply:
I guess they are names of services, so not sure if they would ever be translated though.